tests/installation/ipxe_install.pm

```diff
@@ -103,7 +103,7 @@ sub set_bootscript {
             $cmdline_extra .= " ssh=1 sshpassword=$testapi::password ";    # trigger ssh-text installation
         }
         else {
-            $cmdline_extra .= " vnc=1 VNCPassword=$testapi::password ";    # trigger default VNC installation
+            $cmdline_extra .= " ssh=1 sshpassword=$testapi::password vnc=1 VNCPassword=$testapi::password ";    # trigger default VNC installation
```

Comment:
It was `sshd=1` instead of `ssh=1` before I made https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17400.

Reply:
Of course it has to be, thanks - but then we should also change that for VIDEOMODE=text above I think?

Reply:
No, they are different. we need `ssh=1` rather than `sshd=1` to start an install over ssh console. `sshd=1` will start a sshd service during installation. I have set `sshd=1` in autoyast installation.